org.gnuhealth.MyGNUHealth.yaml

```diff
@@ -0,0 +1,366 @@
+id: org.gnuhealth.MyGNUHealth
```

Comment:
```suggestion
id: org.gnuhealth.mygnuhealth
```

And the file renamed. It's the one used upstream.

Reply:
that was not a suggestion. you should use the same app-id that is used upstream. I case I was unclear.

Reply:
Done